x-pack/plugins/infra/public/containers/logs/log_analysis/api/ml_cleanup.ts

```diff
@@ -8,28 +8,30 @@
 import * as rt from 'io-ts';
 import type { HttpHandler } from '@kbn/core/public';
 
+import { IdFormat, JobType } from '../../../../../common/http_api/log_analysis/v1/id_formats';
```

Comment:
There are many imports referring to `http_api/log_analysis/v1`. We could centralize them in [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/infra/common/http_api/latest.ts) and replace all references to `common/http_api/log_analysis/v1` with `common/http_api/log_analysis/latest`. This would help maintain this feature should the version changes in the future

Reply:
Did this, but didn't touch `/common` since the latest folder depends on the other http_api files